Subject: [PATCH] vold: mount non-primary fuse locations with the multiuser and full write flags

* sdcardfs external sdcards need this or they cannot be properly
  accessed by applications (google music for example)

diff --git a/PublicVolume.cpp b/PublicVolume.cpp
index 893f928..9c136bd 100644
--- a/PublicVolume.cpp
+++ b/PublicVolume.cpp
@@ -211,7 +211,8 @@
                   "-u", "1023", // AID_MEDIA_RW
                   "-g", "1023", // AID_MEDIA_RW
                   "-U", std::to_string(getMountUserId()).c_str(),
-                  "-w",
+                  "-m",
+                  "-w",
                   mRawPath.c_str(),
                   stableName.c_str(),
                   NULL)) {
